create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/__main__.py

```diff
@@ -22,6 +25,11 @@ def main() -> None:
     {%- if cookiecutter.use_uvloop == 'y' %}
     uvloop.install()
     {%- endif %}
+
+    {%- if cookiecutter.use_sentry == 'y' %}
+    init_sentry(app)
+    {%- endif %}
+
```

Comment:
This is simple init function, so move initialize `sentry` to `init_app` from [app.py](https://github.com/aio-libs/create-aio-app/blob/master/create_aio_app/template/%7B%7Bcookiecutter.project_name%7D%7D/%7B%7Bcookiecutter.project_name%7D%7D/app.py#L86-L88)

Insert this function after `init_routes(app)`, please. I guess so this features is not useful for developer mode. As a solution, I recommend provide some value to config file. Some like `debug` with `True/False` or `mode` with `prd/test/dev` properties. (In my opinion `mode` will be more useful because give possibility split different environments) And after that use `sentry` only for production.

Reply:
I usually set dev env and also a specifier of my laptop, so that if errors come from different devs it could be easy to recognize.

Reply:
I don't use sentry so can't  say anything about best usage this library and just proposed.